website/docs/how-to/how-to-add-sso-azure-saml.md

```diff
@@ -0,0 +1,117 @@
+
+## Basic configuration
+
+### Prequisites
+
+This guide expects you to already have:
+
+- Administrator access to the Unleash instance you want to configure
+- Azure AD access for your Azure instance 
+
+### Step 1: Create an Enterprise Application within Azure AD {#step-1}
+
+**a) Sign in to your Azure AD and **create a new Enterprise Application**. 
+
+![In the Azure directory overview, use the add button and select the enterprise application option.](/img/sso-azure-saml-add-enterprise-app.png)
+
+**b) In the Azure AD gallery, select the option to create your own application.**
+
+![The Azure AD gallery. The "create your own application" button is highlighted.](/img/sso-azure-saml-create-own-app.png)
+
+**c) Next, provide the application with a name. When asked what you're looking to do with the application, select the "Integrate any other application you don't find in the gallery (Non-gallery)" option.**
+
+![Azure application creation form. The name is set to "UnleashSSO" and the "non-gallery" option is selected.](/img/sso-azure-saml-name-app.png)
+
+### Step 2: Configure SSO via SAML in Azure AD {#step-2}
+
+**a) On the single sign-on page ("single sign-on" in the side bar), select the "SAML" option**
+
+![Azure: SSO method selection page](/img/sso-azure-saml-saml-choice.png)
+
+**b) Section 1: Basic SAML Configuration {#basic-saml-configuration}**
+
+When configuring SSO with SAML, you'll need to add an **identifier** and a **reply URL**.
+The **identifier** is your Unleash URL. (For hosted instances, that's usually `https://<region>.app.unleash-hosted.com/<instanceName>`).
+
+The **reply URL** is the Unleash callback URL. The Unleash callback URL is available on the Unleash SSO configuration page, and is typically your Unleash URL followed by `/auth/saml/callback`.
+
+![Azure: The basic SAML configuration form with example values filled out for the required fields.](/img/sso-azure-saml-section-one.png)
+
+**c) Section 2: Attributes & Claims {#attributes-and-claims}**
+
+1. Set the "name identifier format" to "Email address".
+2. Select "attribute" as the source.
+3. Enter "user.mail" in the source attribute field.
+
+Optionally, you can also provide a first name and a last name. If provided, these will be used to enrich the data in Unleash.
+
+![Azure: The manage claim form with email configuration filled out](/img/sso-azure-saml-unique-id-email-id.png)
+![Azure: The list of claims used by the SAML setup, including the optional claims for given name and surname](/img/sso-azure-saml-attributes-claim.png)
+
+> Please make sure to replace URLs with the public URL for your Unleash instance. This will require correct region prefix and the instance name. 
+>
+> The correct format is: https://**[region]**.app.unleash-hosted.com/**[instanceName]**/auth/saml/callback
```

Comment:
This should probably be an admonition instead, I think. And what URLs are we asking them to replace?

```suggestion
:::info URLs and formats

Make sure to replace URLs with the public URL for your Unleash instance. This will require correct region prefix and the instance name. 

The correct format is: https://**[region]**.app.unleash-hosted.com/**[instanceName]**/auth/saml/callback

:::
```